README.md

```diff
@@ -2,37 +2,109 @@
 
 ## About
 
-`um2nc-standalone` is an [ACCESS-NRI](https://www.access-nri.org.au/) project to merge multiple versions of unified model conversion tools to a single, canonical project for the ESM1.5 model. 
+`um2nc-standalone` is a `Python3` utility to convert Unified Model data files to NetCDF format. 
+
+The `um2nc-standalone` project is an [ACCESS-NRI](https://www.access-nri.org.au/) initiative to merge multiple versions of unified model conversion tools to a single, canonical project for the ESM1.5 model. 
 
 ## Installation
 
-TODO
+These installation instructions assume deployment to a `Linux` or `MacOS` platform.
 
-* `virtualenv` instructions
-* `conda`/`miniconda`/`micromamba?` instructions
+`Windows` is currently **unsupported**.
 
-## User documentation
+### Downloading `um2netcdf-standalone`
 
-TODO
+```commandline
+cd <your projects dir>
+git clone https://github.com/ACCESS-NRI/um2nc-standalone.git
+cd um2nc-standalone
+```
+
+### Installing Dependencies
+
+#### MacOS
+
+```Bash
+$ brew install udunits
+```
+
+if this fails with some compiler/header errors, try:
```

Comment:
`brew install udunits` ran without reporting any issues. It wasn't until trying `pip install -r requirements.txt` step complained about the missing environment variables:
```
      ValueError: Require to set UDUNITS2_XML_PATH for a cf-units wheel build.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cf-units
Failed to build cf-units
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (cf-units)
```
Do you know if exporting the environment variables should be a required step before installing udunits?

Reply:
Ooops, my earlier cut/paste job of the instructions is deficient!

I originally added `venv` setup notes here:
https://github.com/ACCESS-NRI/amami/issues/13#issue-2247373089

The relevant tweak I needed to get a working `venv` was exporting these env vars:

```Bash
# set up non standard cf-unit paths
export UDUNITS2_XML_PATH=/opt/homebrew/Cellar/udunits/2.2.28/share/udunits/udunits2.xml
export UDUNITS2_LIBDIR=/opt/homebrew/lib
export UDUNITS2_INCDIR=/opt/homebrew/Cellar/udunits/2.2.28/include
```
Let's see if that works...

Reply:
Ah yep, that worked for me! I'm wondering whether this should be included as required before the `brew install udunits` in the install, as the error I got was only raised later when installing the other packages, rather than when installing udunits?

Reply:
This seems to be Python version dependent. I've been trying the instructions for setting up a `conda` environment, and this error only occurs for Py 3.12. Py 3.11 and 3.10 go through without apparent incident.